brave/brave-core
# Removes bind_util

Resolves https://github.com/brave/brave-browser/issues/11471

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- make sure that you can claim promotion
- make sure that you can do contribution (one time tip, recurring tip and ac)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
